.github/workflows/alpine.yml

```diff
@@ -0,0 +1,22 @@
+name: Small TCB builds
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Prepare alpine dockerfile
+      run: echo "From rust:1.72.0-alpine3.18" >> Dockerfile; echo "RUN apk add musl-dev openssl-dev openssl-libs-static protobuf-dev git && git clone https://github.com/intel/ACON.git" >> Dockerfile; echo "RUN cd ACON/acond && CARGO_BUILD_RUSTFLAGS=-Crelocation-model=static cargo build --release --target x86_64-unknown-linux-musl && CARGO_BUILD_RUSTFLAGS=-Ctarget-feature=-crt-static cargo build --release" >> Dockerfile
```

Comment:
Multiple `echo`s can be combined into one `echo -e`.

And shall we use the latest rustc and Alpine?

And the 2 builds will collide on the binary name. Doesn't it matter?

Reply:
Fixed the echo with `echo -e`
Fixed the docker with `From rust:alpine`
Fixed the build process with `cargo clean`